infrastructure/application/services/hasura.ts

```diff
@@ -189,10 +185,10 @@ export const createHasuraService = async ({
         predefinedMetricSpecification: {
             predefinedMetricType: "ECSServiceAverageCPUUtilization",
         },
-        // scale out quickly for responsiveness, but scale in more slowly to avoid thrashing
-        targetValue: 60.0,
+        // scale out early and quickly for responsiveness, but scale in more slowly to avoid thrashing
+        targetValue: 50.0,
```

Comment:
We should scale out a bit sooner/earlier than we were to minimise latencies during the transition.